tests/ocf-url_link_leaking_relative/EPUB/package.opf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Open Container Format</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2022-03-06</dc:date>
+    <dc:description>Use a relative link with several double-dot path segments from the content to a photograph. The folder hierarchy containing the photograph starts at the root level; the number of segments exceeds the height of the folder hierarchy.</dc:description>
+    <dc:identifier id="pub-id">ocf-url_link_leaking_relative</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Potentially leaking relative link</dc:title>
```

Comment:
Perhaps "Relative image reference exceeds depth of hierarchy"? I think it's clearer, from that, what this test is (if you haven't read the markup).

Reply:
ok. I was struggling with this one.